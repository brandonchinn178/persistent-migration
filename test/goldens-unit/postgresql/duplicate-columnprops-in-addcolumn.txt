Left "Duplicate column properties detected: Column {colName = \"age\", colType = SqlInt32, colProps = [NotNull,NotNull]}"