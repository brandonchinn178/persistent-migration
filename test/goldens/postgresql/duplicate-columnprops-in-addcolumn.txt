Left "Duplicate column properties detected: Column {colName = \"age\", colType = SqlInt32, colProps = [Default \"0\",Default \"1\"]}"